rma/i18n/es.po

```diff
@@ -0,0 +1,16 @@
+# Translation of Odoo Server.
```

Comment:
2015-08-05 16:34 GMT-05:00 Pedro M. Baeza <notifications@github.com>:

> Odoo complains in this case? I'm just asking, because I don't know



Yes it returns a WARNING in the log when you update a module without
translations in a lang not installed.

-- 
--------------------
Saludos Cordiales

CEO at Vauxoo <https://www.vauxoo.com> Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
<http://about.me/nhomar?promo=email_sig>